File: src/main/java/no/nav/service/pensjon/regler/to/VilkarsprovForsorgingstilleggRequest.java
package no.nav.service.pensjon.regler.to;

import no.nav.domain.pensjon.regler.krav.Kravhode;

import java.util.Date;

public class VilkarsprovForsorgingstilleggRequest extends ServiceRequest {
    private Kravhode kravhode;
    private Date virkFom;
    private Date virkTom;

}

File: src/main/kotlin/no/nav/pensjon/regler/to/VilkarsprovForsorgingstilleggRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.krav.Kravhode
import java.util.*
import java.time.LocalDate

class VilkarsprovForsorgingstilleggRequest : ServiceRequest() {
    var kravhode: Kravhode? = null
    var virkFomLd: LocalDate? = null
    var virkTomLd: LocalDate? = null
}